spec/javascripts/unit/admin/services/variant_unit_manager_spec.js.coffee

```diff
@@ -23,16 +23,28 @@ describe "VariantUnitManager", ->
 
   describe "unitScales", ->
     it "returns a sorted set of scales for unit type weight", ->
-      expect(VariantUnitManager.unitScales('weight')).toEqual [1.0, 28.35, 453.6, 1000.0, 1000000.0]
+      expect(VariantUnitManager.unitScales('weight')).toEqual [0.001, 1.0, 28.35, 453.6, 1000.0, 1000000.0]
 
     it "returns a sorted set of scales for unit type volume", ->
-      expect(VariantUnitManager.unitScales('volume')).toEqual [0.001, 1.0, 1000.0]
+      expect(VariantUnitManager.unitScales('volume')).toEqual [0.001, 0.01, 0.1, 1.0, 4.54609, 1000.0]
 
   describe "compatibleUnitScales", ->
     it "returns a sorted set of compatible scales based on the scale and unit type provided", ->
       expect(VariantUnitManager.compatibleUnitScales(1, "weight")).toEqual [1.0, 1000.0, 1000000.0]
       expect(VariantUnitManager.compatibleUnitScales(453.6, "weight")).toEqual [28.35, 453.6]
 
+    pending "should load only available unit", ->
```

Comment:
this new test is failing.
I don't know how to update `availableUnits` before running the tests.
I tried directly assigning a new value to `VariantUnitManager.availableUnits`, but it didn't work.